### DISCUSSION TITLE
Editing a transaction unmarks it as reconciled

### DISCUSSION THREAD

**Discussion by pwsoftwaredesigns** [firefly-iii/firefly-iii#7240] [Firefly III] [answered]
Every month, I go through my list of transactions and reconcile them against my bank statement.  Once a transaction is reconciled, a little check mark appears next to it on the list of transactions.  I also scan most of my receipts into the computer but don't always add them as attachments until after I have reconciled the transactions.

I just noticed that, those transactions to which I added attachments are no longer marked as reconciled.  The attachment icon is there, but the check mark is gone. For those transactions that did have an attachment when I reconciled (and thus I didn't edit), both the attachment and check icons are present.

Is this a feature or a bug?

**Comment by JC5** [comment, marked as answer]
Any edit to a reconciled transaction will remove the flag, yes.

**Reply by pwsoftwaredesigns** [reply]
Thanks, that makes sense.

Though I wonder if it would be better to only mark a transaction as unreconciled if an "important" field like: amount, date, etc.. is changed.  All other edits don't change the reconciliation versus a bank statement, so they shouldn't undo that work.
